contracts/RewardController.sol

```diff
@@ -241,6 +241,9 @@ contract RewardController is IRewardController, OwnableUpgradeable {
     * @param _user the account for which the reward is calculated
     */
     function getPendingReward(address _vault, address _user) external view returns (uint256) {
+        if (IHATVault(_vault).rewardControllerRemoved(address(this))) {
+            return 0;
```

Comment:
I guess thsis hould be `return unclaimedReward[_vault][_user]`

Reply:
No because it also calculates the uncommitted reward to be claimed

Reply:
Oh right yes